### PR TITLE
Create network config bootstrapping command - Closes #6073

### LINKED ISSUE #6073: Create network config bootstrapping command
### Description

The network configuration command should generate `json` file and follow the following interface. 

```
lisk generate:network-config <name>

Options:
--label		The application label
--identifier	The Community Identifier of the network 
```

### Motivation

This will enable users to generate multiple configurations to run and test different networks. This will enable user to first be aware of what configurations are available and customize those. 

### Acceptance Criteria

- Output should generate all possible configurations including the default values. 
- The generated json file should be copied to `config/<name>/config.json`.
- It should detected existing configuration with same name and warn user before overwriting

### PR DESCRIPTION
**Pull request by Incede**
### What was the problem?

This PR resolves #6073 

### How was it solved?

* Create network config bootstrapping command

### How was it tested?

* Added unit tests
* Tested manually

### CODE REVIEW

File: commander/src/utils/config.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+export const defaultConfig = {
+	label: 'alpha-sdk-app',
+	version: '0.0.0',
+	networkVersion: '1.1',
```
Comment (shuse2) [inline commander/src/utils/config.ts:4]: i think default can be `1.0`

File: commander/src/commands/generate/network-config.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		// check for existing file at dirName & warn the user of overwriting
+		if (fs.existsSync(configPath)) {
+			const userResponse = await inquirer.prompt({
+				type: 'confirm',
+				name: 'confirm',
+				message:
+					'A config file already exists at the given location. Do you want to overwrite it ?',
+			});
+			if (!userResponse.confirm) {
+				this.error('Aborting');
```
Comment (shuse2) [inline commander/src/commands/generate/network-config.ts:80]:
```suggestion
				this.error('Cancelled to overwrite the config.');
```

File: commander/src/commands/generate/network-config.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		defaultConfig.genesisConfig.communityIdentifier = identifier;
+		// defaultConfig.version = get label from package.json of the app created
+
+		// check for existing file at dirName & warn the user of overwriting
+		if (fs.existsSync(configPath)) {
+			const userResponse = await inquirer.prompt({
+				type: 'confirm',
+				name: 'confirm',
+				message:
+					'A config file already exists at the given location. Do you want to overwrite it ?',
```
Comment (shuse2) [inline commander/src/commands/generate/network-config.ts:77]:
```suggestion
					'A config file already exists at the given location. Do you want to overwrite it?',
```

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: commander/src/commands/generate/network-config.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+		},
+	];
+
+	static flags = {
+		label: flagParser.string({
+			char: 'l',
+			description: 'App Label',
+			default: 'alpha-sdk-app',
+		}),
+		identifier: flagParser.string({
```
[the remark this answers is not in the document]
Comment (Incede) [reply commander/src/commands/generate/network-config.ts:43]: I think `community-identifier` is not allowed as a flag

File: commander/src/commands/generate/network-config.ts [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+import fs from 'fs-extra';
+import { join, resolve } from 'path';
+import inquirer from 'inquirer';
+import { defaultConfig } from '../../utils/config';
+
+export default class NetworkConfigCommand extends Command {
+	static description = 'Creates network configuration file.';
+	static examples = [
+		'generate:network-config mydir',
+		'generate:network-config mydir --label alpha-sdk-app',
```
[the remark this answers is not in the document]
Comment (Incede) [reply commander/src/commands/generate/network-config.ts:26]: not sure what you mean here, can you please elaborate ?

**Review by nazarhussain — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**